ridalot2.0/ridalot2.0/Pages/Feed.razor

```diff
@@ -5,133 +5,132 @@
+            int length = line.Length;
+
+            if (length >= 6 && line.Substring(0, 6) == "Post :")
             {
-                Console.WriteLine(line);
-                img = line.Substring(15);
+
+                n++;
             }
         }
-        return img;
     }
-    private string getDescription()
+    protected override async Task OnInitializedAsync()
     {
+        SomeStartupMethod();
+    }
+
+
+    private string getData(string data, int n)
+    {
+
+        IEnumerable<string> postLines = File.ReadLines("AdInfo.txt").Skip(n * 7);
         foreach (string line in postLines)
         {
-            if (line.Substring(0, 13) == " Description:")
+            int lengthOfString = line.Length;
+            if (lengthOfString >= 15 && line.Substring(0, 15) == data)
+            {
+                img = line.Substring(15);
+                Console.WriteLine(img);
+                return img;
+            }
+            if (lengthOfString >= 13 && line.Substring(0, 13) == data)
             {
-                Console.WriteLine(line);
                 description = line.Substring(13);
-                
+                return description;
+
             }
-        }
-        return description;
-    }
-    private string getHeight()
-    {
-        foreach (string line in postLines)
-        {
-            if (line.Substring(0, 8) == " Height:")
+            if (lengthOfString >= 8 && line.Substring(0, 8) == data)
             {
-                Console.WriteLine(line);
-                height = line.Substring(0);    
+                height = line.Substring(0);
+                return height;
             }
-        }
-        return height;
-    }
-    private string getWidth()
-    {
-        foreach (string line in postLines)
-        {
-            if (line.Substring(0, 7) == " Width:")
+            if (lengthOfString >= 7 && line.Substring(0, 7) == data)
             {
-                Console.WriteLine(line);
                 width = line.Substring(0);
+                return width;
             }
-        }
-        return width;
-    }
-    private string getLength()
-    {
-        foreach (string line in postLines)
-        {
-            if (line.Substring(0, 8) == " Length:")
+            if (lengthOfString >= 8 && line.Substring(0, 8) == data)
             {
-                Console.WriteLine(line);
                 length = line.Substring(0);
+                return length;
             }
-        }
-        return length;
-    }
-    private string getWeight()
-    {
-        foreach (string line in postLines)
-        {
-            if (line.Substring(0, 8) == " Weight:")
+            if (lengthOfString >= 8 && line.Substring(0, 8) == data)
             {
-                Console.WriteLine(line);
                 weight = line.Substring(0);
+                Console.WriteLine(weight);
+                return weight;
             }
+
         }
-        return weight;
+        return "";
```

Comment:
If info was not found, an error should occur telling the user to fix their input. Could be implemented later